networking/src/ApiClient.ts

```diff
@@ -921,6 +926,7 @@ class TeamAnswerParser {
             isChosen,
             isTrickAnswer,
             text,
+            answerContents,
             createdAt,
             updatedAt,
             teamMemberAnswersId,
```

Comment:
At this point, we should cast this to a strongly-typed and parse the json file

Reply:
Yup, sounds good. I will create the parsing in the follow up PR when I start working with the actual answer content.